ci/jobs/management.yml

```diff
@@ -13,14 +13,19 @@ jobs:
       - .: (( inject meta.plan.terraform-bootstrap ))
         params:
           TF_WORKSPACE: management
+          AWS_ROLE_MGT_DEV: arn:aws:iam::((aws_account.management)):role/ci
```

Comment:
Should this be AWS_ROLE_MGT instead of AWS_ROLE_MGT_DEV?

Reply:
it's correct in terms of matching, but the name isn't good, let me rename it